evals/mmmu/mmmu.py

```diff
@@ -1,4 +1,4 @@
-from utils import create_content_and_answers_list, load_and_concatenate_datasets
+from .utils import create_content_and_answers_list, load_and_concatenate_datasets
```

Comment:
With this type of import, when I try to run the eval with:

```bash
inspect eval evals/mmmu/mmmu.py
```

It can't find the import:

```
File "/home/ubuntu/UKGovernmentBEIS/inspect_ai/evals/mmmu/mmmu.py", line 7, in <module>
    from .utils import create_content_and_answers_list, load_and_concatenate_datasets
ModuleNotFoundError: No module named '/home/ubuntu/UKGovernmentBEIS/inspect_ai/evals/mmmu/mmmu'
```

This awkwardness is something we are hoping to resolve _very soon_ (like in the next week) by turning the entire `evals` folder into a Python package.